pkg/inframanager/p4/utils.go

```diff
@@ -49,6 +56,202 @@ type UUIDGenerator struct {
 	internalChan chan uint32
 }
 
+type ActionParam struct {
+	data [][]byte
+}
+
+type KeyData struct {
+	key map[string]client.MatchInterface
+}
+
+// its a common structure to insert and delete entries from table
+type UpdateTable struct {
+	tableName  string
+	mfs        []KeyData
+	actionName string
+	paramData  []ActionParam
+}
```

Comment:
Same comment as before for action. One table can have more than one action.

Reply:
With the current design we can add one more instance of the table with a different action name in json. so each action will have a different instance of UpdateTable struct. 
We can analyze further how it goes when we have a p4info parser, if required can be enhanced further.